README.md

```diff
@@ -44,6 +44,45 @@ Integration with JupyterLab [(Take me to the installation guide!)](https://githu
 
 A PyCharm integration is also in our roadmap so stay tuned!
 
+## 💨 Guide to the Roadmap
+
+### **What is coming up soon?**
+
+After gathering valuable feedback and comments from data scientists, we created the following short-term roadmap for genv:
+
+**1. Environment specific information:** With this feature, you will be able to see the processes, machines and utilization of the machines specifically to your GPU environment when you type nvidia-smi
```

Comment:
The truth is that we won't show utilization specific for the environment so I'd omit this part of the utilization